sdk/python/responsible-ai/mlflow-deployment-with-explanations/requirements.txt

```diff
@@ -0,0 +1,10 @@
+azure-ai-ml
+azure-identity
+azureml-mlflow
+azure-core
+mlflow
+mltable
+ml-wrappers
```

Comment:
where is this requirements.txt used?

Reply:
The dependencies look similar in env.yml. Why can't we merge them?

Reply:
requirements.txt is used to run the notebook; env.yml is used to specify the wrapper model's environment